lib/actions/insert.js

```diff
@@ -26,16 +26,16 @@ function init(cfg) {
     cfg.database
   }${(cfg.domain) ? `?domain=${cfg.domain}&encrypt=${cfg.encrypt}`
     : `?encrypt=${cfg.encrypt}`}`;
-  logger.trace(conString);
+  logger.debug('Connection string is created');
   return co(function* gen() {
     logger.info('Connecting to the database');
     const connection = new cosql.Connection(conString);
     // Always attach an error listener
-    connection.on('error', (err) => this.emit('error', err));
+    connection.on('error', err => this.emit('error', err));
     let sql = cfg.query;
     yield connection.connect();
     logger.info('Connection established');
-    logger.trace('Preparing query=%s', sql);
+    logger.debug('Preparing query...');
     const vars = sql.match(VARS_REGEXP);
     logger.trace('Found following prepared variable:type pairs=%j', vars);
```

Comment:
Do this pairs contains some vulnerable data

Reply:
I tried to run action, in logs:
`Found following prepared variable:type pairs=["@lang","@retweeted:boolean","@favorited:boolean","@text:string","@id:bigint","@created_at:date","@username","@screenname:string"]`
@denyshld @A3a3e1 thoughts? Are column names and types sensitive data?

Reply:
For all table - yes

Reply:
I don't think it is sensitive data